src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -310,6 +356,160 @@ internal override void SetDefaultSourcePath(string defaultPath)
             WriteValueToFile<string>(fileName, "DefaultSourcePath", defaultPath);
         }
 
+#if UNIX
+        /// <summary>
+        /// Gets the identity name to use for writing to syslog.
+        /// </summary>
+        /// <returns>
+        /// The string identity to use for writing to syslog.
+        /// <para>
+        /// The default value is 'powershell'
+        /// </para>
+        /// </returns>
+        internal override string GetSysLogIdentity()
+        {
+            string fileName = Path.Combine(psHomeConfigDirectory, configFileName);
+            string identity = ReadValueFromFile<string>(fileName, "LogIdentity");
+            if
+            (
+                string.IsNullOrEmpty(identity)
+                ||
+                StringComparer.OrdinalIgnoreCase.Compare(LogDefaultValueName, identity) == 0
+            )
+            {
+                identity = "powershell";
+            }
+            return identity;
+        }
+
+        /// <summary>
+        /// Gets the log level filter.
+        /// </summary>
+        /// <returns>One of the PSLevel values indicating the level to log.
+        /// <para>
+        /// The default value is PSLevel.Informational
+        /// </para>
+        /// </returns>
+        internal override PSLevel GetLogLevel()
+        {
+            string fileName = Path.Combine(psHomeConfigDirectory, configFileName);
+            string levelName = ReadValueFromFile<string>(fileName, "LogLevel");
+            PSLevel level;
+            if
+            (
+                StringComparer.OrdinalIgnoreCase.Compare(LogDefaultValueName, levelName) == 0
+                ||
+                !Enum.TryParse<PSLevel>(levelName, true, out level )
+            )
+            {
+                level = PSLevel.Informational;
+            }
+            return level;
+        }
+
+        /// <summary>
+        /// The supported separator characters for listing channels and keywords in configuration.
+        /// </summary>
+        static readonly char[] s_valueSeparators = new char[] {' ', ',', '|'};
```

Comment:
This could go in `System.Management.Automation.Utils.Separators` - it helps reuse to have a common place arrays like this.

Reply:
I don't see any inherit benefit in sharing this constant. It simply introduces unnecessary/unintended coupling.  ON the other hand, if there's an existing pattern that this should follow; I'm happy to look at it.

Reply:
It's more of a meta benefit - folks create arrays like this all over the place, sometimes in a loop.

People copy patterns, so if you follow the pattern, others will too. If you don't, they might follow your pattern - and in the new use case, it might be a bad choice.

Reply:
I've left it encapsulated for now; I consider it a private implementation detail until we have a use case  for sharing it.